lib/views/Delta.js

```diff
@@ -15,79 +15,116 @@ function Delta (module) {
 Delta.prototype.createDeltas = function () {
   this.formating = this.moduleConfig.format || this.moduleConfig['format-options'];
 
-  var yAxes = this.axes.y;
-  var valueAttr = this.axes.y[0].key;
-  var canCalculateDelta = _.some(yAxes, function (y) {
-    return (y.format && y.format.type &&
-      (y.format.type === 'percent' ||
-      y.format.type === 'integer' ||
-      y.format.type === 'number' ||
-      y.format.type === 'currency' ||
-      y.format.type === 'duration'));
-  });
+  var yAxes = this.axes.y,
+      reverseData = false,
+      groupedData = false;
+
+  var period;
+
+  if (this.moduleConfig['data-source'] && this.moduleConfig['data-source']['query-params']) {
+
+    if (this.moduleConfig['data-source']['query-params'].period) {
+      reverseData = true;
 
+      period = this.moduleConfig['data-source']['query-params'].period;
+    }
+
+    if (this.moduleConfig['data-source']['query-params'].group_by && yAxes.length > 1) {
+      _.each(this.moduleConfig['data-source']['query-params'].group_by, function (group) {
+        //more than one group_by?
```

Comment:
Also realised I haven't catered for this yet. We can group by multiple. Going to close PR till this is resolved.